Makefile

```diff
@@ -186,22 +186,22 @@ download_tanzil:
 # 1. list of indexed translations, see update_translations_indexed_list
 # 2. list of offline recitations, see update_recitations_offline_list
 # 3. list of online recitations, see update_recitations_online_list  
-update_post_build:  update_dynamic_resources_postbuild   update_translations_indexed_list update_recitations_offline_list #update_recitations_online_list   
```

Comment:
that's post-build, it list all translations being indexed. its very very bad in performance but necessary

Reply:
@assem-ch I couldn't figure yet what going on

- Each time I build git reported: main.db, information.json, translations.json has been changed.
- So I delete all of them, my reason: generated shouldn't be tracked.
- main.db ok, information.json ok, issue raise up with translation.json.
    It is used in the build before it's got created (no code to create it's empty template).

Reply:
Current behavior, the translations file should be always there since its main resource.. after indexing translations, we run update_translations_indexed_list to get all translations from index and store them in the translation file. current problems involve that the file is essential and also should be tracked , and the operation of listing is very slow. 


Right behavior, no file should be by default, if no file the list should be considered as empty dictionary.  Translations should be added in the indexing phase not after indexing: 
- Index translation x, 
- If success, add translation x to translations.json
- first translation indexed should create the file